articles/ddos-attacks-using-botnets/index.md

```diff
@@ -0,0 +1,72 @@
+
+### Types of DDoS Attacks
+
+#### [Application layer](https://en.wikipedia.org/wiki/Denial-of-service_attack#Application-layer_attacks)
+DDoS attack types include HTTP floods, slow attacks (Slowloris, RUDY), zero-day assaults, and those targeting vulnerabilities in operating systems, web applications, and communication protocols. Comprised of seemingly legitimate and innocent requests with their magnitude usually being measured in requests per second (RPS), the goal of the attack is to overwhelm a target application with requests. This causes high CPU and memory usage that eventually hangs or crashes the application.
+#### [Network layer](https://en.wikipedia.org/wiki/Denial-of-service_attack#HTTP_slow_POST_DoS_attack)
+DDoS attack types include UDP floods, SYN floods, NTP amplification, DNS amplification, SSDP amplification, IP fragmentation, and more. These are high-capacity barrages, measured in gigabits per second (Gbps) or packets per second (PPS). They’re almost always executed by botnets, with a goal of consuming the target’s upstream bandwidth, resulting in network saturation.
+
+![botnet](/engineering-education/ddos-attacks-using-botnets/what-is-a-botnet-and-how-to-protect-yourself-in-2019-1.png.png)<br>
+[Image Source - Safety Detetives](https://www.safetydetectives.com/blog/what-is-a-botnet-and-how-to-protect-yourself-in/)
+
+### Botnets
+A botnet is a collection of computers or other Internet-connected devices that have been infected with malware and now respond to the orders and commands of a central computer, called the Command and Control (C&C) center. Botnets can be used to perform Distributed Denial-of-Service (DDoS) attacks, steal data, send spam, and allow the attacker to access the device and its connection.
+
+The big botnets have a web of millions of devices, and most of the owners have no clue that their devices are compromised. Some are available to rent to the highest bidder, who can use them in whatever way they choose. Often, this means a DDoS attack.
+
+To communicate with a C&C server, the botmaster uses various hidden channels like IRC and HTTP websites. Sometimes, they target popular services like Twitter, Facebook, and even Reddit.
+
+DDoS attacks have become viable (if criminal) and cost-effective options for anyone looking to take out a website or disturb the communication of home and office networks. This results in many attack scenarios, ranging from shady businesses using DDoS to gain a competitive advantage, to assaults in which DDoS becomes a tool of vandalism, revenge, or simply a way to get some attention.
+
+Full-service DDoS attacks are available for as little as $5 per hour, and the interested party can easily stretch their hour with a monthly plan that averages $38. Within the cybercrime ecosystem, botnet DDoS attacks are a mainstream commodity; prices continually drop, while efficacy and sophistication are constantly on the rise.

```

Comment:
Re: the pricing I'm tempted to ask for a citation, but then we also don't want to be linking to such services. Would it be more prudent to say something like "increasingly affordable to the average person" which would also be more timeless?